src/index.js

```diff
@@ -128,7 +143,7 @@ export default async ({ markdownAST }, options = {}) => {
     // highlight the specify lines, if any
     let html = content.div;
     if (query.highlights.length > 0) {
-      const $ = cheerio.load(content.div);
+      const $ = cheerio.load(html);
```

Comment:
I think we can do some optimizations around this area.
Loading html using cheerio is a costly operation, doing it twice is probably just too much.

My suggestion would be:
```js
if (query.highlights.length > 0 || query.lines.length > 0) {
   const $ = cheerio.load(html);
   const file = query.file.replace(/[^a-zA-Z0-9_]+/g, "-").toLowerCase();
   
   // I would do lines first, this actually removes lines from the html
   // no need to highlight lines that are going to be removed anyway =)
   if (query.lines.length > 0) {
     // do removal logic
   }

   if (query.hightlights.length > 0) {
     // do highlighting logic
   }

   html = $.html();
}
```

Reply:
You can move the length counts to variables if you want to - micro-optimization, might not really be needed, but depends on your OCD level - mine is super high 😄)

Reply:
Awesome, have made those optimisations in the latest commit :)